docusign-connector/src/com/axonivy/connector/docusign/event/EnvelopeCompleted.java

```diff
@@ -22,73 +22,73 @@
 import ch.ivyteam.ivy.workflow.query.TaskQuery;
 
 public class EnvelopeCompleted extends AbstractProcessIntermediateEventBean {
-  public static UUID DOCU_SIGN_CLIENT_ID = UUID.fromString("3d3d7114-4397-48c9-8378-931fc35885c3");
+	public static UUID DOCU_SIGN_CLIENT_ID = UUID.fromString("3d3d7114-4397-48c9-8378-931fc35885c3");
```

Comment:
We should not change the formatting of the code for now reason. Makes it really hard to track changes if we added a bug or something in this change..

Please use our formatting prefs: and re-format the sources you changed
[eclipse-preferences.epf.zip](https://github.com/axonivy-market/docusign-connector/files/10740382/eclipse-preferences.epf.zip)
- Unzip this file
- In the Designer: File -> Import -> preferences > select the unpacked .epf
- Restart
- ... then you can apply auto-formatting